src/wizards/connectedap.ts

```diff
@@ -46,6 +51,170 @@ function compareAccessPointConnection(
+      actions.push({ new: { parent: address, element: pVlanPrio } });
+    });
+
+  return actions;
+}
+
+function initGSEElements(
+  doc: XMLDocument,
+  connectedAp: Element
+): SimpleAction[] {
+  const actions: SimpleAction[] = [];
+
+  const iedName = connectedAp.getAttribute('iedName');
+  const apName = connectedAp.getAttribute('apName');
+  const ied = doc.querySelector(
+    `IED[name="${connectedAp.getAttribute('iedName')}"]`
+  );
+
+  const macGenerator = mACAddressGenerator(doc, 'GSE');
+  const appGenerator = appIdGenerator(doc, 'GSE');
+
+  Array.from(ied?.querySelectorAll('GSEControl') ?? [])
+    .filter(gseControl => {
+      const cbName = gseControl.getAttribute('name');
+      const ldInst = gseControl.closest('LDevice')?.getAttribute('inst');
+
+      if (
+        !doc.querySelector(
+          `IED[name="${iedName}"] > AccessPoint[name="${apName}"] > Server, ` +
+            `IED[name="${iedName}"] > AccessPoint[name="${apName}"] > ServerAt`
+        )
+      )
+        return false;
+
+      return !doc.querySelector(
+        `Communication > SubNetwork > ConnectedAP[iedName="${iedName}"] > GSE[cbName="${cbName}"][ldInst="${ldInst}"]`
+      );
+    })
+    .forEach(gseControl => {
+      const cbName = gseControl.getAttribute('name');
+      const ldInst =
+        gseControl.closest('LDevice')?.getAttribute('inst') ?? null;
+
+      const gSE = createElement(connectedAp.ownerDocument, 'GSE', {
+        cbName,
+        ldInst,
+      });
+      actions.push({ new: { parent: connectedAp, element: gSE } });
+
+      const address = createElement(connectedAp.ownerDocument, 'Address', {});
+      actions.push({ new: { parent: gSE, element: address } });
+
+      const pMac = createElement(connectedAp.ownerDocument, 'P', {
+        type: 'MAC-Address',
+      });
+      pMac.textContent = macGenerator();
+      actions.push({ new: { parent: address, element: pMac } });
+
+      const pAppId = createElement(connectedAp.ownerDocument, 'P', {
+        type: 'APPID',
+      });
+      pAppId.textContent = appGenerator();
+      actions.push({ new: { parent: address, element: pAppId } });
+
+      const pVlanId = createElement(connectedAp.ownerDocument, 'P', {
+        type: 'VLANID',
+      });
+      pVlanId.textContent = '000';
+      actions.push({ new: { parent: address, element: pVlanId } });
+
+      const pVlanPrio = createElement(connectedAp.ownerDocument, 'P', {
+        type: 'VLAN-Priority',
+      });
+      pVlanPrio.textContent = '4';
+      actions.push({ new: { parent: address, element: pVlanPrio } });
+
+      const minTime = createElement(connectedAp.ownerDocument, 'MinTime', {
+        unit: 's',
+        multiplier: 'm',
+      });
+      minTime.textContent = '10';
+      actions.push({ new: { parent: gSE, element: minTime } });
+
+      const maxTime = createElement(connectedAp.ownerDocument, 'MaxTime', {
+        unit: 's',
+        multiplier: 'm',
+      });
+      maxTime.textContent = '10000';
```

Comment:
Do you think 1000 might be a better default? I think this would be more typical.

```suggestion
      maxTime.textContent = '1000';
```

Reply:
Yes way not. I have the default from the IEC 61850 system configurator of SIEMENs